CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+* **Provide specific examples to demonstrate the steps**. Include links to files or GitHub projects, or copy/pasteable snippets, which you use in those examples. If you're providing snippets in the issue, use [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+* **Describe the behavior you observed after following the steps** and point out what exactly is the problem with that behavior.
+* **Explain which behavior you expected to see instead and why.**
+* **Include screenshots** which help you demonstrate the steps or point out the part of library which the suggestion is related to.
+* **If you're reporting it crashed**, include a crash report with a stack trace.
+
+### Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion, including completely new features and minor improvements to existing functionality. Following these guidelines helps maintainers and the community understand your suggestion :pencil: and find related suggestions :mag_right:.
+
+#### How Do I Submit A (Good) Enhancement Suggestion?
+
+Enhancement suggestions are tracked as [GitHub issues](https://guides.github.com/features/issues/). Create an enhancement on the repository and provide the following information by filling in [the template](https://github.com/systelab/systelab-components/blob/master/.github/ISSUE_TEMPLATE/feature_request.md).
+
+* **Use a clear and descriptive title** for the issue to identify the suggestion.
+* **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
+* **Provide specific examples to demonstrate the steps**. Include copy/pasteable snippets which you use in those examples, as [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+* **Describe the current behavior** and **explain which behavior you expected to see instead** and why.
+* **Include screenshots** which help you demonstrate the steps or point out the part of library which the suggestion is related to.
+
+### Pull Requests
+
+The process described here has several goals:
+
+- Maintain library's quality
+- Fix problems that are important to users
+- Engage the community in working toward the best possible library
+- Enable a sustainable system for library's maintainers to review contributions
+
+Please follow these steps to have your contribution considered by the maintainers:
+
+1. This repository is aligned with the feature-branch strategy.
+2. Follow all instructions in [the template](PULL_REQUEST_TEMPLATE.md)
```

Comment:
Do we want to use some naming convention for branches? Maybe it is described somewhere else and I didn't find it, sorry. I was thinking in using the number of the enhancement/issue which the branch tries to implement/fix.

Reply:
Yes, it would be better and it's not defined yet. Any proposal? In bitbucket the pattern is "bugfix/feature + IssueID (number) + brief description"

Reply:
I think that the bitbucket pattern can be good enough, so I would suggest just using bugfix-<issueID> or feature-<issueID>. I would not add the description, it can be checked easily by going to the issue and probably will make the branch names very long.